app/config.py

```diff
@@ -39,6 +46,13 @@ class Config(metaclass=MetaFlaskEnv):
     HTTP_SCHEME = "https"
     FRONTEND_HOSTNAME = None
 
+    REDIS_URL = os.getenv("REDIS_URL")
+    REDIS_ENABLED = True
+
+    # 100 requests every 5 minutes
```

Comment:
Comment is now out of date I'm afraid. I think you could even remove this comment entirely if you wanted as I think it doesn't add too much